pkg/network/state.go

```diff
@@ -474,6 +481,37 @@ func (ns *networkState) storeHTTPStats(allStats map[http.Key]*http.RequestStats)
 	}
 }
 
+// storeHTTPStats stores the latest HTTP stats for all clients
+func (ns *networkState) storeHTTP2Stats(allStats map[http.Key]*http.RequestStats) {
```

Comment:
Can we add a test for http2 stats for state?

Reply:
added some tests